File: examples/function/describe/go/describeFunctionStatus.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/function"
)

// Get the info for a deployed function
func DescribeFunctionStatus() {
	functionService := function.NewFunctionService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := functionService.Describe(&function.DescribeRequest{
		Name:    "my-first-func",
		Project: "tests",
	})
	fmt.Println(rsp, err)
}

File: examples/function/describe/node/describeFunctionStatus.js
const { FunctionService } = require("m3o/function");

// Get the info for a deployed function
async function describeFunctionStatus() {
  let functionService = new FunctionService(process.env.MICRO_API_TOKEN);
  let rsp = await functionService.describe({
    name: "my-first-func",
    project: "tests",
  });
  console.log(rsp);
}

describeFunctionStatus();
